src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package service.impl;
+
+import db.Storage;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+import model.Fruit;
+import service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    private static final char NEW_LINE_CHAR = '\n';
+    private static final char SEPARATOR_CHAR = ',';
+    private static final String ERROR_MESSAGE = "Can`t write to file ";
+
+    @Override
+    public void makeStockReportToCsvFile(String outputFileName) {
```

Comment:
this method should only generate your String report it shouldn't write to file. Rename it and delete writing it is the job of separate service